### ISSUE TITLE
cargo vet is busted on mozilla-central

### ISSUE THREAD

**Issue by bholley** [bholley/cargo-vet#42]
STR:

(1) fresh checkout of m-c
(2) `cargo vet init`
(3) `cargo vet`

Results:

```
Vetting Failed!

0 unvetted dependencies:

dependency tree:
  smoosh:0.1.0
    jsparagus:0.1.0
      jsparagus-ast:0.1.0
...
```

**Comment by Gankra** [comment]
Very strange...

**Comment by Gankra** [comment]
fixed in #47
